src/librustdoc/formats/mod.rs

```diff
@@ -40,4 +40,8 @@ impl Impl {
     crate fn trait_did(&self) -> Option<DefId> {
         self.inner_impl().trait_.as_ref().map(|t| t.def_id())
     }
+
+    crate fn def_id(&self) -> clean::ItemId {
```

Comment:
Referring to something as `def_id` when it isn't actually a `DefId` seems bad. I know there are other spots in rustdoc where this is done; those are technical debt, and I'd like to see them go away over time, instead of perpetuating the confusion.

Reply:
Indeed, I'll do that then.